tests/Makefile.am

```diff
@@ -0,0 +1,26 @@
+AM_CFLAGS= -Wall -Werror -I$(top_srcdir)/src $(HWLOC_CFLAGS)
+LDADD = ../src/libmpibind.la libtap.la $(HWLOC_LIBS)
+
+lib_LTLIBRARIES = libtap.la
+libtap_la_SOURCES = tap.c tap.h
+
+TESTS = \
+    error.t \
+    environment.t \
+    coral_lassen.t \
+    epyc_corona.t \
+    coral_ea.t \
+    cts1_quartz.t
+
+check_PROGRAMS = $(TESTS)
+
+TEST_EXTENSIONS = .t
```

Comment:
How is this variable used? Is there anything special about its name?

Reply:
anything listed under check_PROGRAMS is only built when running make check. The same thing can apply to other variables like check_LIBRARIES and check_LTLIBRARIES, etc.

The TEST_EXTENSIONS variable modifies how the log files are named. [Here's a link that explains some of the variables we can use to configure the test harness.](https://www.gnu.org/software/automake/manual/html_node/Parallel-Test-Harness.html)